tests/performance/jit_aggregate_functions_no_key.xml

```diff
@@ -0,0 +1,284 @@
+<test>
+    <preconditions>
+        <table_exists>hits_100m_single</table_exists>
+    </preconditions>
+
+    <settings>
+        <compile_aggregate_expressions>1</compile_aggregate_expressions>
+        <min_count_to_compile_aggregate_expression>0</min_count_to_compile_aggregate_expression>
+    </settings>
+
+    <create_query>
```

Comment:
Why is it a separate file from `jit_aggregate_functions`? Looks like you could have used `group by 1` to put these queries in that file, this would save time on table setup. We use this trick in many queries.